Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
```

Comment:
Not sure how Orbit maps these levels between desktop and mobile, but on wiki, they use semantical level naming.

- Suppressed
- Flat
- Action
- Fixed
- Raised
- Overlay

https://orbit.kiwi/foundation/elevation/